easybuild/easyconfigs/g/GDAL/GDAL-1.10.1-intel-2015b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '1.10.1'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('netCDF', '4.3.2'),
+    ('expat', '2.1.0'),
+    ('libxml2', '2.9.3', '-Python-2.7.10'),
```

Comment:
This is going to result in a lot of head-scratching when this GDAL easyconfig is combined as a dep with something that uses `Python/2.7.11`.

This is true for all dependencies, but `Python` is common enough that this will pop up at some point.

There's a reason we typically include the Python version used by runtime deps in the versionsuffix.